src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -60,8 +60,10 @@ class AddressPage extends Component {
         this.updateAddress = this.updateAddress.bind(this);
         this.onCountryUpdate = this.onCountryUpdate.bind(this);
 
-        const currentCountry = lodashGet(props.privatePersonalDetails, 'address.country') || '';
-        const zipSampleFormat = lodashGet(CONST.COUNTRY_ZIP_REGEX_DATA, [currentCountry, 'samples'], '');
+        const currentCountry = lodashGet(this.props.privatePersonalDetails, 'address.country') || '';
+        const currentCountryISO = PersonalDetails.getCountryISO(currentCountry);
+        const targetCountryISO = !_.isEmpty(currentCountryISO) ? currentCountryISO : CONST.COUNTRY.US;
```

Comment:
Could we combine these two lines like so? 

```suggestion
        const currentCountryISO = PersonalDetails.getCountryISO(currentCountry) || CONST.COUNTRY.US;
```